modules/ROOT/pages/distributed-file-polling.adoc

```diff
@@ -0,0 +1,23 @@
+= Distributed File Polling
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: file polling, cluster
+
+*_Enterprise Edition_*
+
+Some connectors, such as the File connector or FTP connector, regularly poll directories and read certain files as they are created in the directories polled. These files may reside on a remote file system, including file systems of nodes belonging to a Mule High Availability (HA) Cluster.
```

Comment:
avoid "may". Not sure what to recommend. I assume they sometimes (but not always) reside on a remote file system? If sometimes, I'll check to see what our suggested replacement is. I'd normally recommend "can" but need to see if that rec should change.

Also,  isn't it only the On New File operation in the File or FTP connector that actually polls regularly in Mule 4?

Reply:
@aiannucci Can this be confirmed?

Reply:
You can say that the connectors poll. Just more precise that it's the operation.

Reply:
Updated